src/modules/featured/featured.service.ts

```diff
@@ -36,7 +36,7 @@ export class FeaturedService {
         this.redisClient,
         cacheKey,
         getAssetLiked,
```

Comment:
now that we invalidate the cache we could use a higher ttl like oneDay or mode

Reply:
Done